.buildkite/pipeline.yml

```diff
@@ -277,6 +277,29 @@ steps:
   # BitBar tests
   #
 
+  - label: 'BitBar web browser test - Chrome latest'
+    depends_on: "push-branch-cli"
+    timeout_in_minutes: 10
+    plugins:
+      artifacts#v1.9.0:
+        upload: "maze_output/failed/**/*"
+      docker-compose#v4.7.0:
+        pull: browser-tests
+        run: browser-tests
+        service-ports: true
+        use-aliases: true
+        verbose: true
+        command:
+          - "--farm=bb"
+          - "--browser=chrome_latest"
+          - "--aws-public-ip"
+          - "--fail-fast"
+    env:
+      RUBY_VERSION: "2.7"
+    concurrency: 5
+    concurrency_group: 'bitbar-web'
+    concurrency_method: eager
+
```

Comment:
I'd be happy to remove the `BitBar web browser test - Firefox v107` step now.  The thing it's really testing is general support for browser testing on BitBar, as opposed to specific browser versions.